android/gradletealeaf.xml

```diff
@@ -1,9 +1,10 @@
 //<!--START_PLUGINS_DEPENDENCIES-->
 	//<!--Facebook Dependencies-->
-    implementation 'com.facebook.android:facebook-core:4.34.0'
-    implementation 'com.facebook.android:facebook-login:4.34.0'
-    implementation 'com.facebook.android:facebook-places:4.34.0'
-    implementation 'com.facebook.android:facebook-messenger:4.34.0'
-    implementation 'com.facebook.android:facebook-applinks:4.34.0'
-    implementation 'com.facebook.android:facebook-marketing:4.34.0'
+    implementation 'com.facebook.android:facebook-core:4.41.0'
+    implementation 'com.facebook.android:facebook-login:4.41.0'
+    implementation 'com.facebook.android:facebook-places:4.41.0'
+    implementation 'com.facebook.android:facebook-messenger:4.41.0'
+    implementation 'com.facebook.android:facebook-applinks:4.41.0'
+    implementation 'com.facebook.android:facebook-marketing:4.41.0'
+    implementation 'com.facebook.android:facebook-share:4.41.0'
```

Comment:
will we need all of these @vivekb91 ?

Reply:
I will check, All these may no the needed